File: test-server.py
import http.server
import socketserver

PORT = 5555

class MyHandler(http.server.SimpleHTTPRequestHandler):
    def do_GET(self):
        self.send_response(200)
        self.send_header('Content-type', 'text/html')
        self.end_headers()
        self.wfile.write(b'<h1>Python Server Funcionando!</h1><p>Se ves isto, o problema NAO e Windows.</p>')

with socketserver.TCPServer(("127.0.0.1", PORT), MyHandler) as httpd:
    print(f"Servidor Python rodando em http://127.0.0.1:{PORT}")
    print("Ctrl+C para parar")
    httpd.serve_forever()

File: test-server.js
const http = require('http');

const server = http.createServer((req, res) => {
  console.log('📥 Request recebido:', req.url);
  res.writeHead(200, { 'Content-Type': 'text/html' });
  res.end('<h1>Teste - Servidor Node funcionando!</h1><p>Se vês isto, Node.js está OK.</p>');
});

server.listen(5555, '127.0.0.1', () => {
  console.log('✅ Servidor teste rodando em http://127.0.0.1:5555');
  console.log('Pressiona Ctrl+C para parar');
});

server.on('error', (err) => {
  console.error('❌ Erro no servidor:', err);
});

// Keep alive
setInterval(() => {
  console.log('💓 Servidor ainda ativo...');
}, 5000);
